compiler/src/dune

```diff
@@ -4,3 +4,21 @@
 (menhir
  (flags "--table" "--explain")
  (modules parser))
+
+(library
+  (name jasmin_support)
+  (wrapped false)
+  (modules annotations fInfo iInfo location syscall_t)
```

Comment:
Would it make sense to put these files used at extraction time in a separate folder?

Reply:
The hardest would be to find a name for this folder.

Reply:
I don't know, something like "coq_stubs"/"extraction_stubs"/"coq_glue"?